docs: Small ADR-003 changes (#141)
umee-network/umee: docs/architecture/ADR-003-borrow-assets.md

> ### Positive

Reviewer: ```suggestion
### Positive

```

- uTokens used as collateral increase in base asset value in the same way that lend positions do. This counteracts borrow position interest.
- UX of enabling/disabling token types as collateral is simpler than depositing specific amounts

> - lengthPrefixed(borrowerAddress) | tokenDenom key pattern facilitates sorting open positions by user

Reviewer: ```suggestion
- `lengthPrefixed(borrowerAddress) | tokenDenom` key pattern facilitates getting open borrow positions by account address.
```

> ### Negative

Reviewer: ```suggestion
### Negative

```

- `x/bank` module must be extended to prohibit peer-to-peer transfers of uTokens when they would violate `x/leverage` borrowing limit.

> - lengthPrefixed(borrowerAddress) | tokenDenom key pattern makes it more difficult to sort all open positions by token type.

Reviewer: ```suggestion
- `lengthPrefixed(borrowerAddress) | tokenDenom` key pattern makes it more difficult to get all open borrow positions by token denomination.
```
